app/database/models/member.py

```diff
@@ -2,7 +2,14 @@
 
 
 class Member(Document):
-    gid   = LongField(required=True)
-    uid   = LongField(required=True, primary_key=True, unique_with=["gid"])
-    xp    = LongField(required=True)
-    level = LongField(required=True)
+    gid           = LongField(null=False, required=True)
+    uid           = LongField(null=False, required=True, primary_key=True, unique_with=["gid"])
+    xp            = LongField(default=0)
+    level         = LongField(default=0)
+    display_name  = StringField()
+    name          = StringField()
+    discriminator = LongField()
```

Comment:
Do we really need to store the user's display name, username, and discriminator? That information can get obsolete very fast, and the effort to get the information from Discord isn't that hard (you typically will have a DiscordMember object already)

Reply:
The name was to visually identify a record, but I agree with you that it is currently bloat. I'll remove those